### PR TITLE
#133 multiline input factory

### PR DESCRIPTION
**Pull request by kovalandrew**

### CODE REVIEW

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import dev.icerock.moko.widgets.style.view.TextHorizontalAlignment
+import dev.icerock.moko.widgets.style.view.TextStyle
+import dev.icerock.moko.widgets.style.view.TextVerticalAlignment
+import dev.icerock.moko.widgets.style.view.WidgetSize
+
+expect class MultilineInputViewFactory(
+    cornerRadiusValue: CornerRadiusValue?,
+    borderColor: Color?,
+    borderWidth: Float?,
+    backgroundViewColor: Color?,
```
Comment (Alex009) [inline widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt]: set default = null please

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    private val borderWidth: Float?,
+    private val backgroundViewColor: Color?,
+    private val margins: MarginValues?,
+    private val padding: PaddingValues?,
+    private val textStyle: TextStyle?,
+    private val labelTextColor: Color?,
+    private val textHorizontalAlignment: TextHorizontalAlignment?
+) : ViewFactory<InputWidget<out WidgetSize>> {
+
+    private var isPlaceholderShow: Boolean = false
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt]: factory used by multiple inputs, so you can't store nothing in factory class

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/utils/TextStyleExt.kt [hunk 3]
```diff
@@ -90,3 +91,19 @@ fun CATextLayer.applyTextStyleIfNeeded(textStyle: TextStyle?) {
         )
     }
 }
+
+fun UITextView.applyTextStyleIfNeeded(textStyle: TextStyle?) {
+    if (textStyle == null) return
+
+    val currentFontSize = font?.pointSize ?: UIFont.systemFontSize
+    val styleSize = textStyle.size?.toDouble()
+    val styleStyle = textStyle.fontStyle
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/utils/TextStyleExt.kt]: replace to `fontStyle` from `styleStyle` :)

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt (hunk 2 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt]: Will move it implementation to TextViewDelegate

Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt]: Moved

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/utils/TextStyleExt.kt (hunk 3 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/utils/TextStyleExt.kt]: Replaced here and in previous ext methods

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt (hunk 1 above)
Comment (kovalandrew) [reply widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/MultilineInputViewFactory.kt]: added

**Pull request merged**